src/50_Visual_Effects/Basics_of_orientation_effects.html

```diff
@@ -0,0 +1,304 @@
+<!--
```

Comment:
Does it make sense to enable a standalone preview?

```
<!---{
    "preview": "default"
}--->
```

You can separate samples there by adding headings in between that don't have a comment (and hence won't be shown in the doc view).

Reply:
It seems like the "View Demos" option doesn't show up on mobile devices and since this component will be mainly used on Mobile devices, I don't think a standalone preview adds much value.

Reply:
View Demo does show (it's a bug if not). Only the playground is disabled on mobile.

Reply:
Oops. Sorry I missed this comment. 

I remember running a search for this code and didn't see the `View Demo` button in the Chrome device emulator. I don't remember what page that was. Can't repro that now. Added the preview mode.